crates/topos-tce/Cargo.toml

```diff
@@ -18,6 +18,7 @@ tracing-subscriber = { workspace = true, default-features = false,  features = [
 tracing.workspace = true
 
 tce_transport = { package = "topos-tce-transport", path = "../topos-tce-transport" }
+topos-uci = { path = "../topos-uci" }
```

Comment:
You don't need `topos-uci` here, `topos-core` is already imported. Just `use topos-core::uci`